src/sentry/api/serializers/models/organization_member/base.py

```diff
@@ -67,8 +67,9 @@ def serialize(
             "email": obj.get_email(),
             "name": obj.user.get_display_name() if obj.user else obj.get_email(),
             "user": attrs["user"],
-            "role": obj.role,
-            "roleName": roles.get(obj.role).name,
+            "role": obj.role,  # Deprecated, use orgRole instead
+            "roleName": roles.get(obj.role).name,  # Deprecated
+            "orgRole": obj.role,
```

Comment:
Should there be an `orgRoleName`?

Reply:
It's not hard to do a `orgRoleList.find(r => r.id == id)`; on the frontend. 

IMO there should be some helper method to get the org/team role on the frontend, but refactoring all the places that uses `roleName` is outside the scope for now.